sim/battle.js

```diff
@@ -972,7 +972,7 @@ class Battle extends Dex.ModdedDex {
 	getRelevantEffectsInner(thing, callbackType, foeCallbackType, foeThing, bubbleUp, bubbleDown, getAll) {
 		if (!callbackType || !thing) return [];
 
-		let validCallbackType = !!getAll || CALLBACK_TYPES.has(callbackType);
+		let validCallbackType = !!getAll || this.callbackTypes.has(callbackType);
```

Comment:
`getAll` kind of ruins the naming here (and forces us to bail out of the optimization). In both cases, its so niche that it doesn't seem worth drawing attention to, and `getAllOrValidCallbackType` is a bit of a mouthful. Still, for clarity's sake I'd be open to a rename.